tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckCLDR.java

```diff
@@ -169,7 +169,13 @@ public StatusAction getShowRowAction(
             ) {
 
             PathHeader.SurveyToolStatus status = ph.getSurveyToolStatus();
-            // always forbid deprecated items - don't show.
+            /*
+             * Always forbid DEPRECATED items - don't show.
+             *
+             * Currently, bulk submission and TC voting are allowed even for SurveyToolStatus.HIDE,
+             * but not for SurveyToolStatus.DEPRECATED. If we ever want to treat HIDE and DEPRECATED
+             * the same here, then it would be simpler to call ph.shouldHide which is true for both.
```

Comment:
I think it's an oversight that bulk submission allows `SurveyToolStatus.HIDE` and that `FORBID_READONLY` should be returned for such items. What do you think @macchiati ? Could be a separate item / bug because it's not part of progress bars.